secretmanager/secretmanager_test.go

```diff
@@ -107,7 +108,14 @@ func testCleanupSecret(tb testing.TB, name string) {
 }
 
 func testIamUser(tb testing.TB) string {
-	return "user:sethvargo@google.com"
+	tb.Helper()
+
+	v := os.Getenv("GOLANG_SAMPLES_SERVICE_ACCOUNT_EMAIL")
+	if v == "" {
+		tb.Fatalf("testIamUser: missing GOLANG_SAMPLES_SERVICE_ACCOUNT_EMAIL")
```

Comment:
Please skip instead. Tests should ideally pass if someone clones the repo and runs the tests.

Reply:
Done